### ISSUE TITLE
Docker Image Format v1 and Docker Image manifest version 2

### ISSUE THREAD

**Issue by vstrefezzi** [docker-library/ruby#452]
After upgrading Docker-CE to v26.0.2 I can no longer pull the `ruby:2.2.4` image

```
$ docker pull ruby:2.2.4    
2.2.4: Pulling from library/ruby
[DEPRECATION NOTICE] Docker Image Format v1 and Docker Image manifest version 2, schema 1 support is disabled by default and will be removed in an upcoming release. Suggest the author of docker.io/library/ruby:2.2.4 to upgrade the image to the OCI Format or Docker Image manifest v2, schema 2. More information at https://docs.docker.com/go/deprecated-image-specs/

```

Is this a known issue? Am I missing something?

Running Ubuntu 22.04 with docker-ce 26.0.2

```
$ docker version
Client: Docker Engine - Community
 Version:           26.0.2
 API version:       1.45
 Go version:        go1.21.9
 Git commit:        3c863ff
 Built:             Thu Apr 18 16:27:07 2024
 OS/Arch:           linux/amd64
 Context:           default

Server: Docker Engine - Community
 Engine:
  Version:          26.0.2
  API version:      1.45 (minimum version 1.24)
  Go version:       go1.21.9
  Git commit:       7cef0d9
  Built:            Thu Apr 18 16:27:07 2024
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.6.31
  GitCommit:        e377cd56a71523140ca6ae87e30244719194a521
 runc:
  Version:          1.1.12
  GitCommit:        v1.1.12-0-g51d5e94
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
```

**Comment by vstrefezzi** [comment]
Fixed this pulling other version ruby:2.2.10 (latest release)

**Comment by tianon** [comment]
Yeah, unfortunately this is very expected -- `ruby:2.2.4` is 8 years old (2016), and Moby, containerd, and even Docker Hub are working to stop supporting the older schema1 image format they're using (https://github.com/docker/roadmap/issues/173).
